docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/palette-helm-ref.md

```diff
@@ -31,6 +31,40 @@ information, refer to the [Image Swap Configuration](#image-swap-configuration)
 
 :::
 
+### Global
+
+The global block allows you to provide configurations that apply globally to the installation process.
+
+### Image Pull Secret
+
+The `imagePullSecret` block allows you to provide image pull secrets that will be used to authenticate with private
+registries to obtain the images required for Palette installation. This is relevant if you have your own mirror
+registries you use for Palette installation.
+
+| **Parameters**     | **Description**                                                                                                                                                                                                                                                                         | **Type** | **Default value** |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ----------------- |
+| `create`           | Specifies whether to create a secret containing credentials to your own private image registry.                                                                                                                                                                                         | Boolean  | `false`           |
+| `dockerConfigJson` | The `config.json` file created when you successfully log into your image registry in base64 encoded format on a single line. For more information about the `config.json` file, refer to [Kubernetes Documentation](https://kubernetes.io/docs/concepts/containers/images/#config-json) | String   | None              |
```

Comment:
```suggestion
| `dockerConfigJson` | The **config.json** file value containing the registry URL and credentials for your image registry in base64 encoded format on a single line. For more information about the **config.json** file, refer to [Kubernetes Documentation](https://kubernetes.io/docs/concepts/containers/images/#config-json) | String   | None              |
```

@lennessyy I updated the description for `dockerConfigJson` to more accurately reflect what the file is. The successful log into the registry line, while true, is a bit different in Kubernetes versus a Docker context. I'm also afraid someone may think they need to log in from inside the k8s cluster which could cause confusion 😓
Feel free to adjust as needed.